lib/validators/url_validator.rb

```diff
@@ -24,16 +24,51 @@ def validate_each(record, attribute, value)
         end
 
         record.errors[attribute] << options[:message] unless value =~ regexp
```

Comment:
I'll move the regexp check to his own method, that will be called inside the `catch` block.

The regexp allow us to check for proper domains (it reject domains like `http://google` and `http://google.looooongltd`). But this regexp doesn't contemplate other authority options like user/pass or port, and I think we'll need to change